# HLA Class I and II assimilation
Brief background (current literature and essential reference(s)):

The major histocompatibility complex (MHC) or human leukocyte antigen system in humans is a set of highly polymorphic genes located on chromosome 6 involved in immune regulation and cellular differentiation. The genes encode specialised cell surface proteins able to present antigenic peptides and recognise foreign pathogens or antigens and trigger an immune response. HLA molecules are divided into two classes; Class I and Class II MHC molecules. Class I molecules, encoded by genes HLA-A, HLA-B and HLA-C are transmembrane glycoproteins that consist of an alpha heavy chain and a beta-microglobulin, whereas class II molecules present on antigen presenting cells, are encoded by genes HLA-DR, HLA-DQ and HLA-DP and similar to class I are heterodimers but consist of two homogenous peptides consisting of an alpha and a beta chain.

HLA genes are inherited as a haplotype on chromosome 6. As transplantation of any foreign tissue can induce cellular or humoral immune responses, HLA matching between recipient and donor is crucial in kidney transplantation to minimise these responses and avoid graft rejection if possible[1].

The high rate of polymorphism in HLA genes has resulted in the need for a comprehensive nomenclature system. Each HLA allele is given a specific unique number corresponding to up to four sets of digits separated by colons[2]. Initially, HLA alleles were named based on serological recognition which resulted in alleles assigned a number after the gene (HLA-A23 or A23) which now corresponds to a low resolution typed allele. High resolution typed alleles are named using two sets of digits which define a specific DNA sequence of the antigen binding site. Additional digits can represent the presence of a synonymous mutation in the sequence and/or the presence of a mutation outside the coding region [2]. 

Current practice involves low resolution typing for recipients and donors of solid organ transplants based on mismatch calculation for HLA-A, HLA-B and HLA-DR alleles. This is calculated by the presence or absence of same type low alleles between patients rather than matching at a high-resolution level. There have been studies [3,4,5] that suggest that HLA matching could be performed based on the number and physiochemical properties of amino acid polymorphisms of HLA alleles rather than the current practice of binary matching. In order to allow for this, high-resolution typing is essential. As all HLA genes are present in the same locus of chromosome 6, they are inherited as a haplotype allowing for prediction of common HLA gene associations. 

As part of an on-going NIHR study led by a transplant surgeon in Addenbrooke’s Hospital and in collaboration with the Tissue Typing department, this project aims to use historical low-resolution tissue typing data and convert them into high-resolution alleles based on common associations and population frequencies for Caucasian individuals. In turn, the high-resolution data can then be used in an algorithm able to calculate the electrostatic differences between HLA alleles and assign a score that can be used in mismatch calculations.





Aims and Objectives:

•	using population frequencies predict high resolution HLA alleles from existing low resolution data
•	recalculate historical mismatches using high-resolution data
•	assimilated data to be used  to enable calculation of mismatching based on electrostatic differences between HLA alleles

